<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentModuleWidget</class>
 <widget class="QWidget" name="StudentModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>801</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string notr="true">选课/退课</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>771</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>课程搜索：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_search">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(255, 255, 255, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(0, 255, 127, 200);
font: 11pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_search">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="button_enter_course">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
          </property>
          <property name="text">
           <string>选课</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_exit_course">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
          </property>
          <property name="text">
           <string>退课</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_entered">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
          </property>
          <property name="text">
           <string>已选课程</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView_course">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string notr="true">课程成绩</string>
    </attribute>
    <widget class="QTableView" name="tableView_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>568</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_id">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>班级：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_classno">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
